crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -795,6 +796,19 @@ export namespace parse_output {
     createTreeCursor(): cursor.Cursor;
   }
 }
+export namespace query {
+  export interface QueryResult {
+    queryNumber: number;
+    bindings: { [key: string]: cursor.Cursor[] };
+  }
+  export class Query {
+    static parse(text: string): Query;
```

Comment:
I wonder if `text` can be confused with Solidity "queried" text here. What about `expression`?

Reply:
I don't think it will be confused. The method is `Query::parse` after all.